src/mixins/popupMixin.js

```diff
@@ -213,14 +232,17 @@ export default {
     },
     $hide () {
       if (this.isShown()) {
-        clearTimeout(this.timeoutId)
-        removeClass(this.$refs.popup, SHOW_CLASS)
-        this.timeoutId = setTimeout(() => {
-          removeFromDom(this.$refs.popup)
-          this.timeoutId = 0
-          this.$emit('input', false)
-          this.$emit('hide')
-        }, this.transitionDuration)
+        clearTimeout(this.hideTimeoutId)
+        this.hideTimeoutId = setTimeout(() => {
+          removeClass(this.$refs.popup, SHOW_CLASS)
+          // gives fade out time
+          setTimeout(() => {
```

Comment:
There might be a bug: while you clearing `hideTimeoutId`, if the inner code have already run, the inner timeout will not be cleared. and it will still be removed from DOM while time arrived.

Reply:
hi @wxsms , you are right. Please check out my last commit https://github.com/wxsms/uiv/pull/267/commits/7578092ed70fd4801f08d95c032f6de8293dd2d5

Thanks :)